pkgs/misc/sagetex/default.nix

```diff
@@ -29,10 +29,8 @@ stdenv.mkDerivation (finalAttrs: rec {
     cp -va *.sty *.cfg *.def "$path/"
   '';
 
-  passthru = {
-    tlType = "run";
-    pkgs = [ finalAttrs.finalPackage ];
-  };
+  # for texlive.combine, insist that this is a "tex" output, even if it is "out"
+  passthru.tlOutputName = "tex";
```

Comment:
Uff, this is not enough yet – `texlive.combine` really wants to see the `.tex` attribute. Very annoying that `mkDerivation` errors out `outputs` does not contain `"out"` or `"dev"`, otherwise `outputs = [ "tex" ]"` would solve the problem.